uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -1130,11 +1165,11 @@ sub registerScanner {
 
 =head3 createNewCandID($dbhr)
 
-Creates a new CandID.
+Creates a new C<CandID>.
 
 INPUT: database handle reference
 
-RETURNS: (int) CandID
+RETURNS: (int) C<CandID>
```

Comment:
maybe you can put integer in the end (I remember one of the earlier codes had this exact same format and you change the order

Reply:
done